### ISSUE TITLE
Type Error when one panel mode is enabled

### ISSUE THREAD

**Issue by slavashvets** [coderaiser/cloudcmd#116]
* **Version** (`cloudcmd -v`):  v6.15.0
* **Node Version** `node -v`: v8.0.0
* **OS** (`uname -a` on Linux): Linux 879c6c8fa55b 3.10.0-514.el7.x86_64 #1 SMP Thu Nov 3 17:15:05 PDT 2016 x86_64 GNU/Linux
* **Browser name/version**: Chrome 58.0.3029.110 (64-bit)
* **Used Command Line Parameters**: Default docker params
* **Changed Config**: `"onePanelMode": true,`

## Steps to reproduce
1. Run docker image (v6.15.0)
2. Enable "One Panel Mode"

## Expected behavior
One panel is working with no errors.

## Actual behavior
One panel works, but buttons don't work. Error:

```javascript
Uncaught (in promise) TypeError: Cannot read property 'querySelector' of null
    at Object.t.exports.getByDataName (http://docker/cloudcmd.js:1:55873)
    at Object.t.exports.setOnPanel (http://docker/cloudcmd.js:1:47037)
    at g (http://docker/cloudcmd.js:1:31410)
    at r (http://docker/cloudcmd.js:1:1654)
    at Function.r.series (http://docker/cloudcmd.js:1:2683)
    at http://docker/cloudcmd.js:1:2707
    at http://docker/cloudcmd.js:1:31246
    at g.(anonymous function).then.g.(anonymous function) (http://docker/cloudcmd.js:1:9837)
    at <anonymous>
```

**Comment by coderaiser** [comment]
Thank you, fixed with 6706d359.

**Comment by coderaiser** [comment]
Landed in [v6.15.1](https://github.com/coderaiser/cloudcmd/releases/tag/v6.15.1).
